lib/hammer_cli_katello/activation_key.rb

```diff
@@ -219,11 +219,9 @@ class ProductContentCommand < HammerCLIKatello::ListCommand
         from :content do
           field :id, _("ID")
           field :name, _("Name")
-          field :type, _("Type")
-          field :contentUrl, _("URL")
-          field :gpgUrl, _("GPG Key")
```

Comment:
Unless those items are completely useless to the user, we should leave them since users may already be used to using them.